android/src/main/java/com/swmansion/gesturehandler/core/HoverGestureHandler.kt

```diff
@@ -70,6 +71,10 @@ class HoverGestureHandler : GestureHandler<HoverGestureHandler>() {
       return true
     }
 
+    if (handler is RNGestureHandlerRootHelper.RootViewGestureHandler) {
```

Comment:
We can also specify that `pointerType` has to be `mouse`, but I haven't found any issue with `stylus`
```suggestion
    if (handler is RNGestureHandlerRootHelper.RootViewGestureHandler && pointerType == POINTER_TYPE_MOUSE) {
```